src/libs/actions/Policy.ts

```diff
@@ -2746,6 +2746,59 @@ function createPolicyCategory(policyID: string, categoryName: string) {
     API.write(WRITE_COMMANDS.CREATE_WORKSPACE_CATEGORIES, parameters, onyxData);
 }
 
+function createPolicyTag(policyID: string, tagName: string) {
+    const onyxData: OnyxData = {
+        optimisticData: [
+            {
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: `${ONYXKEYS.COLLECTION.POLICY_TAGS}${policyID}`,
+                value: {
+                    Tag: {
```

Comment:
I don't think this is accurate. A user can rename the tagList name and this request would fail. We need to dynamically update this name. Same goes for the other cases below

Reply:
Any idea how to get that name? We can have multiple items in a tag list. How would that be handled?

Reply:
Moving [this](https://expensify.slack.com/archives/C02NK2DQWUX/p1710280280510949?thread_ts=1710198549.235919&cid=C02NK2DQWUX) discussion over to Slack.

Reply:
Handled!